Polyhedron/demo/Polyhedron/Plugins/PCA/Scene_edit_box_item.cpp

```diff
@@ -232,7 +235,7 @@ struct Scene_edit_box_item_priv{
     //Fragment source code
     const char fragment_source[] =
     {
-      "#version 120 \n"
+      "//#version 100  \n"
```

Comment:
In the end it's only for OpenGL ES 2.0 that I will remove the #version, because the standard is #version 100 es and I had errors saying es was unexpected. Plus in my Android version, none of my shader had a #version, so I guess it will be ok.